reddit-background

```diff
@@ -410,6 +429,10 @@ def _handle_cli_options(desktops):
     set_verbosity(args.verbose)
     if args.download_only:
         set_download_only(True)
+
+    if args.download_only and args.number_of_images:
```

Comment:
Subtle bug here, not that it should matter too much, but would still be good to fix anyway.

If I do `--number-of-image=0`, I'd expect this essentially be a NOP.

However, since 0 is Falsey, we won't honor the CLI parameter and will end up downloading an image anyway.

A slightly reworked version would have the default of the CLI param be `None`, and then you'd test whether you'd set the value with `args.number_of_images is not None`.

Since NUMBER_OF_IMAGES global defaults to `1`, we should be good.